### PR TITLE
Refreshes the reports and fixes a few smaller problems

### PR DESCRIPTION
**Pull request by rschwietzke**

### CODE REVIEW

**Comment by rschwietzke** [comment]
Contains:

* Chart area is cleaner now
* Little more margin and padding for freshes look
* Different size of font for tables, larger mostly, will adjust to smaller numbers when the display is smaller (1200px breakpoint)
* Fixes missing WebP support for trendreports
* Fixes failing trendreport creation due to the change of the date format (no msec anymore for start and end)
* Error listing has more room for traces now, table has been sized differently

File: config/reportgenerator.properties [hunk 1]
```diff
@@ -11,7 +11,7 @@
 ## Whether to create links from the error entries in the load test report to the
 ## corresponding result browsers in the results directory. Valid values are
 ## "false" (default) and "true".
-#com.xceptance.xlt.reportgenerator.linkToResultBrowsers = true
+com.xceptance.xlt.reportgenerator.linkToResultBrowsers = true
```
Comment (jowerner) [inline config/reportgenerator.properties:14]: Was this change committed intentionally or by accident?

**Comment by rschwietzke** [comment]
> Empty tables show a "No data available" row. The CSS classes attached to such rows are sometimes contradicting, for example `value text no-data` means to align the text right/left/center. Okay to reduce this to `no-data`?
Yes. Some of the orignal stuff is just a little unaligned, so yes, this makes sense to change.

**Comment by jowerner** [comment]
In the page header, the link in "Created with Xceptance LoadTest x.x.x" has lost its link decoration. Furthermore, the spacing between "LoadTest" and "x.x.x" appears to be gone.

**Comment by jowerner** [comment]
In the page footer, the link in "Copyright (c) 2005-2023 Xceptance Software Technologies GmbH" has lost its link decoration and is no longer shown in white color.

File: config/xsl/loadreport/sections/errors.xsl [hunk 2]
```diff
@@ -62,16 +62,16 @@
 							</div>
 						</xsl:when>
 						<xsl:otherwise>
-							<p class="text-center">
-								There are no charts to show.
+							<p class="text-center no-data">
```
Comment (jowerner) [inline config/xsl/loadreport/sections/errors.xsl:65]: Outside tables, `no-data` does currently not provide any specific styling.

File: config/xsl/loadreport/sections/errors.xsl [hunk 3]
```diff
@@ -216,20 +216,18 @@
...
 											</a>
 										</div>
 									</div>
 								</xsl:for-each>
 							</div>
 						</xsl:when>
 						<xsl:otherwise>
-							<p class="text-center">
-								There are no charts to show.
+							<p class="text-center no-data">
```
Comment (jowerner) [inline config/xsl/loadreport/sections/errors.xsl:229]: Outside tables, `no-data` does currently not provide any specific styling.

Comment (rschwietzke) [reply config/xsl/loadreport/sections/errors.xsl:229]: Fixed. no-data is not a p with styling.

**Comment by rschwietzke** [comment]
> In the page footer, the link in "Copyright (c) 2005-2023 Xceptance Software Technologies GmbH" has lost its link decoration and is no longer shown in white color.

Same color as header. Link color hovering is back and with the same color as well.

**Comment by rschwietzke** [comment]
> In the page header, the link in "Created with Xceptance LoadTest x.x.x" has lost its link decoration. Furthermore, the spacing between "LoadTest" and "x.x.x" appears to be gone.

Fixed.

**Comment by jowerner** [comment]
There are two more tables with custom "no data" messages:

* Load Profile Settings: "There is no load profile defined?!?"
* Custom Agent JVM Settings: "There are no custom settings."

Want to replace these with "No data available" as well?

**Comment by jowerner** [comment]
> > In the page footer, the link in "Copyright (c) 2005-2023 Xceptance Software Technologies GmbH" has lost its link decoration and is no longer shown in white color.
> 
> Same color as header. Link color hovering is back and with the same color as well.

Hm, the link text color is still default, i.e. blue. Only when hovering over it, turns the color to white.

**Comment by rschwietzke** [comment]
> There are two more tables with custom "no data" messages:
> 
> * Load Profile Settings: "There is no load profile defined?!?"
> * Custom Agent JVM Settings: "There are no custom settings."
> 
> Want to replace these with "No data available" as well?
Sure.

**Pull request merged**